### PR TITLE
[NOJIRA]: remove del and replace with rimraf

### PR DESCRIPTION
**Pull request by mungodewar**
Closes PR: https://github.com/Skyscanner/bpk-svgs/pull/1

Instead of updating this package to ESM which would be very costly for downstream consumers we are opting to remove `del` and replace with `rimraf` - an alternative deletion package.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ import fs from 'fs';
 import punycode from 'punycode';
 
 import del from 'del';
```
Comment (arnaugm) [inline gulpfile.babel.js:22]: shouldn't this import be removed? why is linter not complaining?

Comment (mungodewar) [reply gulpfile.babel.js]: Yeah the linter didn't catch it :( Thanks Arnau - import removed now.

**Review by arnaugm — APPROVED** [review]
Do you know why linter didn't catch it?

**Comment by mungodewar** [comment]
> Do you know why linter didn't catch it?

Locally I don't know. In the pipeline the other issue of the registry was stopping it getting to the linting step.

**Pull request merged**